### PR TITLE
recipe for pymssql

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Related to #3136 #3133

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
closing and reopening to trigger build?

**Comment by mariusvniekerk** [comment]
ping @ericdill

File: recipes/pymssql/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+  # fn: {{ name }}-{{ version }}.tar.gz
+  # url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  # sha256: {{ sha256 }}
+  git_url: https://github.com/pymssql/pymssql.git
+  git_rev: c3b445b2f7dd1b9d1ee8028879795f7671a8f86f
+
+build:
+  number: 0
+  skip: true  # [win]
```
Comment (ericdill) [inline recipes/pymssql/meta.yaml]: if you're skipping all windows builds why bother with all the features?

File: recipes/pymssql/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{% set name = "pymssql" %}
+{% set version = "2.2.0pre1" %}
```
Comment (ericdill) [inline recipes/pymssql/meta.yaml]: do we know that 2.2.0 is going to be the next release?

Comment (ericdill) [reply recipes/pymssql/meta.yaml]: If not this should be 2.1.3.post# where # is how many commits after 2.1.3 `c3b445b2f7dd1b9d1ee8028879795f7671a8f86f` is

**Comment by ericdill** [comment]
@mariusvniekerk are you satisfied with skipping the windows build for now?

**Comment by mariusvniekerk** [comment]
Yeah I'll fix that once feedstocks are live

**Pull request merged**